shared/constants/teams.tsx

```diff
@@ -107,6 +107,19 @@ export const makeEmailInviteError = I.Record<Types._EmailInviteError>({
   message: '',
 })
 
+export const teamRoleWithBotsByEnum = ((m: {[K in Types.MaybeTeamRoleTypeWithBots]: RPCTypes.TeamRole}) => {
+  const mInv: {[K in RPCTypes.TeamRole]?: Types.MaybeTeamRoleTypeWithBots} = {}
+  for (const roleStr in m) {
+    // roleStr is typed as string; see
+    // https://github.com/facebook/flow/issues/1736 .
```

Comment:
this is a flow issue, I'm confused as to why you're running into it. Are you sure it's the same problem? If I'm reading the above correctly I think it should be a `RPCTypes.TeamRole` but these types are complicated. You probably want an `as` rather than a ts ignore, though, if you do have to ignore it.

Reply:
preexisting in `teamRoleByEnum` below

Reply:
either way we should probably fix rather than leave ignored forever? but if it's too hard i guess the next person to touch this code can try again.